modules/phase_field/src/kernels/ACBarrierFunction.C

```diff
@@ -31,7 +31,8 @@ ACBarrierFunction::ACBarrierFunction(const InputParameters & parameters)
     _dmudvar(getMaterialPropertyDerivative<Real>("mu", _uname)),
     _d2mudvar2(getMaterialPropertyDerivative<Real>("mu", _uname, _uname)),
     _vname(getParam<std::vector<VariableName>>("v")),
-    _d2mudvardeta(_n_eta)
+    _d2mudvardeta(_n_eta),
+    _eta_map(getParameterJvarMap("v"))
```

Comment:
I'd rather call this `_vmap`. The multiple different names (`v`, `_vals`, eta) get a bit confusing. Otherwise this looks good (once you fix indentation)